scss/6-components/_navigation_greedy-nav.scss

```diff
@@ -51,6 +52,10 @@
       }
     }
 
+    span {
```

Comment:
a conscious choice so as not to have styling classes in the GreedyNav typescript